integration-tests/bats/config.bats

```diff
@@ -208,4 +208,30 @@ teardown() {
 
     # cleanup
     dolt config --global --unset init.defaultBranch
-}
\ No newline at end of file
+}
+
+@test "config: default init branch is not master" {
```

Comment:
The test above this one is now wrong, it sets the default branch to its now-default value. Change it to use master